practice.md

```diff
@@ -49,6 +50,13 @@ You can then initialize a new 18F repository's license information with:
 
 It's also recommended to copy and paste [this paragraph for the end of your README](https://github.com/18F/open-source-policy/blob/master/README.md#public-domain) that sums up what's going on.
 
+### Accepting contributions from the public
+
+Any 18F project can accept open source contributions from the public. Contributors must agree to release their contributions into the international public domain.
```

Comment:
OK @benbalter, in 2d5037f I attempted to be more precise, while ultimately delegating responsibility to projects in non-standard situations to figure it out for themselves.